Patch to removed the build dependency to vcpkg as we are using EasyBuild for that
author: J. Sassmannshausen
diff --git a/healthgps.orig/CMakePresets.json b/healthgps/CMakePresets.json
index d29ef60..b3a3132 100644
--- a/healthgps.orig/CMakePresets.json
+++ b/healthgps/CMakePresets.json
@@ -6,9 +6,9 @@
       "description": "Shared configuration presets",
       "hidden": true,
       "generator": "Ninja",
-      "binaryDir": "${sourceDir}/out/build/${presetName}",
+      "binaryDir": "${sourceDir}/build",
       "cacheVariables": {
-        "CMAKE_INSTALL_PREFIX": "${sourceDir}/out/install/${presetName}"
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/install/${presetName}"
       }
     },
     {
@@ -18,15 +18,7 @@
       "inherits": "base",
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Debug",
-        "CMAKE_CXX_COMPILER": "g++",
-        "CMAKE_TOOLCHAIN_FILE": {
-          "value": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
-          "type": "FILEPATH"
-        }
-      },
-      "vendor": {
-        "microsoft.com/VisualStudioSettings/CMake/1.0": { "hostOS": [ "Linux" ] },
-        "microsoft.com/VisualStudioRemoteSettings/CMake/1.0": { "sourceDir": "$env{HOME}/.vs/$ms{projectDirName}" }
+        "CMAKE_CXX_COMPILER": "g++"
       }
     },
     {
@@ -108,4 +100,4 @@
       "configurePreset": "x64-debug"
     }
   ]
-}
\ No newline at end of file
+}
